### ISSUE TITLE
A+ All results table (course points page) should not include unconfirmed points in the normal official points

### ISSUE THREAD

**Issue by Mankro** [apluslms/a-plus#941]
The All results table includes unconfirmed points in the normal points, but it should not. It must be possible to separate normal real points from the unconfirmed and unofficial points. It must also be possible to combine normal and unconfirmed points without unofficial points.

The table should have checkboxes for selecting
1. normal points
2. normal with unconfirmed points
3. normal with unofficial points
4. normal with unconfirmed and unofficial

In other words, it should NOT combine the options "unofficial" and "unconfirmed" into a single checkbox. It must be possible to separate unconfirmed points from unofficial.

Unconfirmed points are part of the mandatory feedback feature, which is triggered by the category setting `confirm_the_level` (Mandatory activity).

Related to #918.

**Comment by lainets** [comment]
I want to confirm that I have understood the intended behaviour of `confirm_the_level`. If some exercise is in a category with `confirm_the_level` set to true, all submissions to itself and its sibling exercises (they share a parent) are considered unconfirmed until a passing submission is made to it. E.g. if exercises A, B and C are in the same chapter and C's category has `confirm_the_level = True`, all points from A, B and C are unconfirmed until a submission is made to C that gets at least `points_to_pass` points. This doesn't confirm the points of the child exercises of A, B and C (if such exist). Is this correct?

**Comment by Mankro** [comment]
> I want to confirm that I have understood the intended behaviour of `confirm_the_level`. If some exercise is in a category with `confirm_the_level` set to true, all submissions to itself and its sibling exercises (they share a parent) are considered unconfirmed until a passing submission is made to it. E.g. if exercises A, B and C are in the same chapter and C's category has `confirm_the_level = True`, all points from A, B and C are unconfirmed until a submission is made to C that gets at least `points_to_pass` points. This doesn't confirm the points of the child exercises of A, B and C (if such exist). Is this correct?

Yes, correct. The O1 course uses `confirm_the_level`. The feedback questionnaires at the end of chapters are mandatory. Giving feedback "confirms" the points of the exercises in the same chapter.
